Generate GitHub connector from OpenAPI spec (#288)
ballerina-platform/module-ballerinax-github: docs/open-api-spec/Sanitations.md

# Introduction

> This file records the sanitation done on top of the official OAS. These sanitation's are done for improving usability and as workaround for known limitations in language side.

Reviewer: ```suggestion
This file records the sanitation done on top of the official OAS. These sanitation's are done for improving the usability and, as workarounds for some of the known limitations in the language side.
```

> 1. Remove resource paths,

Reviewer: ```suggestion
1. Removed the below resource paths
```

    * /users/{userId}/settings - This path has around 20 odd sub paths and does not add significant usability.
    * /users/{userId}/watch & /users/{userId}/stop - This will be covered in Google PubSub

> 2. Rename `Thread` schema to `NotificationThread`. This is done as a workaround for issue, [Openapi tool does not escape in built symbols](https://github.com/ballerina-platform/ballerina-standard-library/issues/5067)

Reviewer: ```suggestion
2. Renamed `Thread` schema to `NotificationThread`. This is done as a workaround for issue, [Openapi tool does not escape in built symbols](https://github.com/ballerina-platform/ballerina-standard-library/issues/5067)
```

> 2. Rename `Thread` schema to `NotificationThread`. This is done as a workaround for issue, [Openapi tool does not escape in built symbols](https://github.com/ballerina-platform/ballerina-standard-library/issues/5067)

Reviewer: Are these the only sanitizations we have done? I see only 2 changes here.

Author: Actually no, This is just the template, lemme push the changes to the Sanitization.md now
